mysql-test/suite/rocksdb/t/use_direct_reads_writes.test

```diff
@@ -44,4 +53,47 @@ shutdown_server 10;
 --source include/wait_until_connected_again.inc
 --disable_reconnect
 
---exec grep "enable both use_direct_io_for_flush_and_compaction" $MYSQLTEST_VARDIR/log/mysqld.1.err | cut -d] -f2
+--let $assert_text= Checking for use_direct_io_for_flush_and_compaction
+--let $assert_file= $MYSQLTEST_VARDIR/log/mysqld.1.err
+--let $assert_only_after= use_direct_reads_writes.test restart direct_writes
+--let $assert_select= enable both use_direct_io_for_flush_and_compaction
+--let $assert_count= 1
+--source include/assert_grep.inc
+
+# Verify invalid direct-writes and --rocksdb_flush_log_at_trx_commit combination at startup fails
+--exec echo "wait" >$_expect_file_name
+shutdown_server 10;
+
+# Add a delimiter to the log file
+--exec echo "[Note] use_direct_reads_write.test restart flush_log" >> $MYSQLTEST_VARDIR/log/mysqld.$_server_id.err
+
+--exec echo "restart:--rocksdb_flush_log_at_trx_commit=1 --rocksdb_allow_mmap_writes=1" >$_expect_file_name
+--sleep 0.1
```

Comment:
Not sure this is not racy. @georgelorchpercona, what you ended up doing for this kind of test?

Reply:
I believe we just used restart_mysqld which waits for the wait_until_connected to timeout. Makes the test run longer but incurs no resource overhead while waiting. Here is a similar example that we fixed recently https://github.com/percona/percona-server/blob/5.7/mysql-test/suite/rocksdb/t/use_direct_reads_writes.test#L10

Reply:
Actually, I just remembered, this is the test that we were both confused with in PS because PS would still start even if the plugin failed to initialize. So, in our case, we just start the server then grep the error log for the complaint, then stop the server and continue on. So, our (PS) is going to be different from FB-MySQL in this test behavior just like the use_direct_reads_writes.test